### PR TITLE
added code for web statistics

### LINKED ISSUE #196: Gather statistics on Web access
Add ability to count the access to http://www.devopsloft.io

### PR DESCRIPTION
**Pull request by ynixon**
Fix #196

### CODE REVIEW

File: web_s2i/templates/layout.html [hunk 1]
```diff
@@ -35,6 +35,24 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous"></script>
 
+    <!-- Default Statcounter code for Devopsloft  -->
+    <script type="text/javascript">
+    var sc_project=11982438;
+    var sc_invisible=0;
+    var sc_security="03b2cd4a";
+    var scJsHost = "https://";
```
Comment (lioramilbaum) [inline web_s2i/templates/layout.html]: does it mean the site is https://www.deveoploft.io? we haven't implemented it yet.

Comment (ynixon) [reply web_s2i/templates/layout.html]: yes

Comment (ynixon) [reply web_s2i/templates/layout.html]: no, sorry - I wrote done http:// ..

Comment (lioramilbaum) [reply web_s2i/templates/layout.html]: I am not following

Comment (ynixon) [reply web_s2i/templates/layout.html]: https is to the site statcounter.com
I implemented the following URL:  http://www.devopsloft.io/

Comment (ynixon) [reply web_s2i/templates/layout.html]: I defined the following URL as open for the public
do you want me to add this to the readme?

Comment (lioramilbaum) [reply web_s2i/templates/layout.html]: I don't think it is required in the README. But, that is my 2 cents. Maybe others think differently.

Comment (ynixon) [reply web_s2i/templates/layout.html]: I have just googled and found recommendation for having different **statcounter** project for each environment.
Any idea what is the recommended way to pass parameters to the app per environment ?

Comment (lioramilbaum) [reply web_s2i/templates/layout.html]: You want the app to "know" in which environment it is running?

Comment (ynixon) [reply web_s2i/templates/layout.html]: Yes, 
So we will not mix statistics among environment.
Currently any web access is counted, even dev

Comment (lioramilbaum) [reply web_s2i/templates/layout.html]: We know the IP address of the PROD environment - 52.10.238.106. Can you use it?

Comment (ynixon) [reply web_s2i/templates/layout.html]: I assume that we can. the application.py can know by IP that it is PROD.
but I prefer to have statistics also for stage and dev - separately 
This way we will see how much people used our code :)

Comment (lioramilbaum) [reply web_s2i/templates/layout.html]: Ohh, do you want to collect stats on how many DEV/STAGE/CI environment spinned. Let me think a bit.

Comment (lioramilbaum) [reply web_s2i/templates/layout.html]: I think, Vagrant can inject an environment variable to the VM and from the VM to the running app container.

**Comment by lioramilbaum** [comment]
@ynixon when we find a way to tell the application the environment name than we can expose it in the app home page.

**Comment by ynixon** [comment]
WIP

**Comment by ynixon** [comment]
> @ynixon when we find a way to tell the application the environment name than we can expose it in the app home page.

Now you have variable: `os.getenv('ENVIRONMENT')` in `application.py` to add it to the home page (a future PR)

**Comment by lioramilbaum** [comment]
@ynixon exposing the environment name in the application home page footer is ready in my fork on branch show-env-name. LMK if you want me to submit a PR on this one.

File: .travis.yml [hunk 2]
```diff
@@ -13,6 +13,9 @@ addons:
 install:
 - pip3 install -r tests/requirements-test.txt
 before_script:
+- echo ENVIRONMENT=dev >> .env
```
Comment (lioramilbaum) [inline .travis.yml]: ENVIRONMENT should be CI

**Comment by lioramilbaum** [comment]
LGTM

**Pull request merged**